paper-tab.html

```diff
@@ -36,6 +37,7 @@
 `--paper-tab` | Mixin applied to the tab | `{}`
 `--paper-tab-content` | Mixin applied to the tab content | `{}`
 `--paper-tab-content-unselected` | Mixin applied to the tab content when the tab is not selected | `{}`
+`--paper-font-common-base` | Font mixin | Defined in `paper-styles/typography.html`
```

Comment:
Yeah, I think what goes in that docs section is "mixins this element has defined", not "mixins this element has used". For example, we don't mention all the flexbox ` --layout-inline` and friends mixins we're using either.